src/main/java/com/zaxxer/sansorm/SqlClosureElf.java

```diff
@@ -32,11 +32,11 @@ private SqlClosureElf()
      * @param type The type of the desired object.
      * @param ids The ID or IDs of the object.
      * @param <T> The type of the object.
-     * @return The object or <code>null</code>
+     * @return The object or {@code null}
      */
     public static <T> T getObjectById(Class<T> type, Object... ids)
     {
-        return SqlClosure.execute(c -> OrmElf.objectById(c, type, ids));
+        return SqlClosure.sqlExecute(c -> OrmElf.objectById(c, type, ids));
```

Comment:
"execute" is deprecated. however i dunno why, old name seems reasonable

Reply:
That change was to provide more clarity in code.  When using static imports, both in Java and even more so languages like Kotlin, the generic nature of a method name like ``execute()`` can impede readability.

For example, we have a lot of code that executes SQL in an ``Executor`` (e.g. java.util.concurrent.Executor), and when combined with an *import static*, e.g.

``import static com.zaxxer.SqlClosureElf.execute``

you end up with code that looks like this:
```java
class Something extends ThreadPoolExecutor {
   ...
   private void methodOne() {
      execute(() -> {
         // some async code
      });
   }

   private void methodTwo() {
      execute(con -> {
         // some sql code
      });
   }

   private void methodThree() {
      execute(() -> {
         execute(con -> {
             // some async sql
         }
      });
}
```
It ends up reading better with the change.  For example:
```java
private void methodThree() {
   execute(() -> {
      sqlExecute(con -> {
      }
   }
}
```
And even more so in Kotlin, which does away with some of the boilerplate of Java:
```kotlin
private fun methodThree() = execute { sqlExecute { con ->
      // some async sql
   }
}
```
(In Kotlin ``execute { execute { con ->...``) looks *really* weird, even though correct). 😬